ui-kit/PrayerCard/PrayerCard.js

```diff
@@ -25,7 +25,12 @@ const PrayerCard = (props = {}) => {
             </Box>
           )}
           {props?.coverImage && (
-            <Avatar src={props?.coverImage} maxWidth="20%" />
+            <Avatar
+              src={props?.coverImage}
+              objectFit="cover"
```

Comment:
If this CSS property is already in the styled component, do we need to replicate this property here?

Reply:
Good catch, not sure why that happened lol. I removed it